data/json/recipes/recipe_ammo.json

```diff
@@ -177,12 +177,8 @@
   ],
   "components" : [
     [
-      [ "superglue", 1 ],
-      [ "bone_glue", 1 ],
-      [ "thread", 20 ],
-      [ "yarn", 20 ],
-      [ "sinew", 20 ],
-      [ "plant_fibre", 20 ]
+      [ "adhesive", 1, "LIST" ],
```

Comment:
Duct tape is part of the adhesive requirement but it's in this recipe already with the fletching part of the recipe. Does that have any effect on components needed to make the arrow? Last time I checked it caused some wonky behavior with how much of that component was used.

Reply:
Duct tape will be displayed twice in the components list: 

![default](https://user-images.githubusercontent.com/16213433/35534745-81498ba4-0552-11e8-928b-58349251be77.png)

If you have multiple components for one of the LIST - you will be asked which alternative to use:

![default](https://user-images.githubusercontent.com/16213433/35534910-0018f294-0553-11e8-9be5-8fb91c3f6540.png)

![default](https://user-images.githubusercontent.com/16213433/35534916-036118fa-0553-11e8-9912-f840382a2eee.png)

Components are spent correctly.